spring-batch-integration/src/main/java/org/springframework/batch/integration/config/annotation/EnableBatchIntegration.java

```diff
@@ -75,15 +84,60 @@
  *       		.build();
  * 	}
  *
+ * 	// Middleware beans omitted
+ *
  * }
  * </pre>
  *
+ * For remote partitioning, an example of a configuration class would be:
+ *
+ * <pre class="code">
+ * &#064;Configuration
+ * &#064;EnableBatchIntegration
+ * &#064;EnableBatchProcessing
+ * public class RemotePartitioningAppConfig {
+ *
+ * 	&#064;Autowired
+ * 	private RemotePartitioningMasterStepBuilderFactory masterStepBuilderFactory;
+ *
+ * 	&#064;Autowired
+ * 	private RemotePartitioningWorkerStepBuilderFactory workerStepBuilderFactory;
+ *
+ * 	&#064;Bean
+ * 	public Step masterStep() {
+ *       	 return this.masterStepBuilderFactory
+ *       		.get("masterStep")
+ *       		.partitioner("workerStep", partitioner())
+ *       		.gridSize(GRID_SIZE)
+ *       		.outputChannel(outgoingRequestsToWorkers())
+ *       		.inputChannel(incomingRepliesFromWorkers())
```

Comment:
Sounds like would be better to name them as `request` and `reply` channels. 
I don't tell that this `master/worker` approach reminds me [Scatter-Gather](https://docs.spring.io/spring-integration/docs/5.0.6.RELEASE/reference/html/messaging-routing-chapter.html#scatter-gather) with an appropriate channel names 😄